doc/api/async_context.md

```diff
@@ -136,6 +136,50 @@ changes:
 Creates a new instance of `AsyncLocalStorage`. Store is only provided within a
 `run()` call or after an `enterWith()` call.
 
+### Static method: `AsyncLocalStorage.bind(fn)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `fn` {Function} The function to bind to the current execution context.
+
+Binds the given function to the current execution context.
+
+### Static method: `AsyncLocalStorage.snapshot()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Returns a callback that captures the current async context and invokes a
```

Comment:
The description is not really clear to me.
I think we should more clear describe that async context is captured at `snapshot()` time and the returned function (I would not name it callback here) invokes the given function forwarding any given arguments but not `thisArg` using the captured async context.